lib/config.js

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+
+var config = require('bedrock').config;
+
+// configure event logging module
+config['event-log'].eventTypes.AgreementAccept = {
```

Comment:
We are doing NounVerb with other events.  Like CredentialAccept and CredentialReject.  What would you like to do here?